cluster-up/cluster/k8s-provider-common.sh

```diff
@@ -19,4 +19,17 @@ function up() {
 
     # Make sure that local config is correct
     prepare_config
+
+    # Activate cluster-network-addons-operator if flag is passed
+    if [ "$KUBEVIRT_WITH_CNAO" == "true" ]; then
+        kubectl="${_cli} --prefix $provider_prefix ssh node01 -- sudo kubectl --kubeconfig=/etc/kubernetes/admin.conf"
+
+        $kubectl create -f /tmp/cnao/namespace.yaml
+        $kubectl create -f /tmp/cnao/network-addons-config.crd.yaml
+        $kubectl create -f /tmp/cnao/operator.yaml
+        $kubectl wait deployment -n cluster-network-addons cluster-network-addons-operator --for condition=Available --timeout=200s
```

Comment:
Is Available condition available on a Deployment?

Reply:
It has this conditions:
```yaml
status:
  availableReplicas: 1
  conditions:
  - lastTransitionTime: "2020-03-03T16:03:12Z"
    lastUpdateTime: "2020-03-03T16:03:12Z"
    message: Deployment has minimum availability.
    reason: MinimumReplicasAvailable
    status: "True"
    **type: Available**
  - lastTransitionTime: "2020-03-03T16:02:47Z"
    lastUpdateTime: "2020-03-03T16:03:12Z"
    message: ReplicaSet "cluster-network-addons-operator-69d7cd44f5" has successfully
      progressed.
    reason: NewReplicaSetAvailable
    status: "True"
    type: Progressing
  observedGeneration: 1
  readyReplicas: 1
  replicas: 1
  updatedReplicas: 1

```

Reply:
:man_facepalming: I was blind.